packages/pieces/community/PDF/src/lib/actions/parse-pdf-from-url.ts

```diff
@@ -0,0 +1,23 @@
+import { createAction, Property } from '@activepieces/pieces-framework';
+import { readPdfText } from 'pdf-text-reader';
+
+export const parsePdfFromUrl = createAction({
+  // auth: check https://www.activepieces.com/docs/developers/piece-reference/authentication,
+  name: 'parsePdfFromUrl',
+  displayName: 'Parse text from PDF URL',
+  description: 'Downloads PDF file from internet and parses the text',
+  props: {
+    'file-url': Property.ShortText({
```

Comment:
@nyamkamunhjin Sorry for my bad explanation, Property.File does accept urls too, out of the box too. 

So no need for this action.